components/schemes/Shape/tests/unit/MDCShapeSchemeTests.m

```diff
@@ -68,4 +69,20 @@ - (void)testInitWithMaterialDefaults {
                                                                 andSize:4]);
 }
 
+- (void)testShapeCategoryCopy {
+  // Given
+  MDCShapeCategory *cat = [[MDCShapeCategory alloc] initCornersWithFamily:MDCShapeCornerFamilyCut
```

Comment:
Nit: Please name this something like "shapeCategory" or "testCategory" or "originalCategory" or even "category". A cat is also an animal and it confused me for a moment when I read the code.